src/test/kotlin/org/rust/lang/core/completion/RsFragmentSpecifierCompletionTest.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.lang.core.completion
+
+import org.intellij.lang.annotations.Language
+import org.rust.lang.core.macros.FragmentKind
+
+class RsFragmentSpecifierCompletionTest : RsCompletionTestBase() {
```

Comment:
Let's add a test that checks that we don't complete specifiers right after `$` sign, i.e. `$/*caret*/`